NuGet.Config

```diff
@@ -7,10 +7,11 @@
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption/nuget/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
As per https://learn.microsoft.com/en-us/nuget/concepts/security-best-practices#nuget-feeds guidance, we can't have nuget.org as package source.

Reply:
I understand and was expecting this, but in order to test this it needs this for now, otherwise the NuGet package cannot be found in those other resources.